sass.py

```diff
@@ -107,8 +107,16 @@ def from_lambda(cls, name, lambda_):
         :rtype: :class:`SassFunction`
 
         """
-        argspec = inspect.getargspec(lambda_)
-        if argspec.varargs or argspec.keywords or argspec.defaults:
+        if PY3:
+            argspec = inspect.getfullargspec(lambda_)
+            varargs, varkw, defaults = (argspec.varargs, argspec.varkw,
```

Comment:
`kwonlyargs` is not supported on `getargspec`, it's something that was introduced with `getfullargspec`. I'm not entirely clear what this function is used for, I just went for feature-parity. What would `kwonlyargs` add over `varkw`?